hack/update-codegen-clients.sh

```diff
@@ -27,58 +27,60 @@ BOILERPLATE_HEADER="$( pwd )/hack/boilerplate/boilerplate.go.txt"
-  --trim-path-prefix github.com/kcp-dev/kcp
+# bash "${CODEGEN_PKG}"/generate-groups.sh "deepcopy" \
+#   github.com/kcp-dev/edge-mc/third_party/conditions/client github.com/kcp-dev/edge-mc/third_party/conditions/apis \
+#   "conditions:v1alpha1" \
+#   --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate/boilerplate.generatego.txt \
+#   --output-base "${SCRIPT_ROOT}" \
+#   --trim-path-prefix github.com/kcp-dev/edge-mc
 
-bash "${CODEGEN_PKG}"/generate-groups.sh "deepcopy,client" \
-  github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/client github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/apis \
-  "wildwest:v1alpha1" \
-  --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate/boilerplate.generatego.txt \
-  --output-base "${SCRIPT_ROOT}" \
-  --trim-path-prefix github.com/kcp-dev/kcp
+# bash "${CODEGEN_PKG}"/generate-groups.sh "deepcopy,client" \
+#   github.com/kcp-dev/edge-mc/test/e2e/fixtures/wildwest/client github.com/kcp-dev/edge-mc/test/e2e/fixtures/wildwest/apis \
+#   "wildwest:v1alpha1" \
+#   --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate/boilerplate.generatego.txt \
+#   --output-base "${SCRIPT_ROOT}" \
+#   --trim-path-prefix github.com/kcp-dev/edge-mc
 
-pushd ./test/e2e/fixtures/wildwest/apis
-${CODE_GENERATOR} \
-  "client:outputPackagePath=github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/client,apiPackagePath=github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/apis,singleClusterClientPackagePath=github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/client/clientset/versioned,headerFile=${BOILERPLATE_HEADER}" \
-  "lister:apiPackagePath=github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/apis,headerFile=${BOILERPLATE_HEADER}" \
-  "informer:outputPackagePath=github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/client,singleClusterClientPackagePath=github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/client/clientset/versioned,apiPackagePath=github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/apis,headerFile=${BOILERPLATE_HEADER}" \
-  "paths=./..." \
-  "output:dir=./../client"
-popd
+# pushd ./test/e2e/fixtures/wildwest/apis
+# ${CODE_GENERATOR} \
+#   "client:outputPackagePath=github.com/kcp-dev/edge-mc/test/e2e/fixtures/wildwest/client,apiPackagePath=github.com/kcp-dev/edge-mc/test/e2e/fixtures/wildwest/apis,singleClusterClientPackagePath=github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/client/clientset/versioned,headerFile=${BOILERPLATE_HEADER}" \
+#   "lister:apiPackagePath=github.com/kcp-dev/edge-mc/test/e2e/fixtures/wildwest/apis,headerFile=${BOILERPLATE_HEADER}" \
+#   "informer:outputPackagePath=github.com/kcp-dev/edge-mc/test/e2e/fixtures/wildwest/client,singleClusterClientPackagePath=github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/client/clientset/versioned,apiPackagePath=github.com/kcp-dev/edge-mc/test/e2e/fixtures/wildwest/apis,headerFile=${BOILERPLATE_HEADER}" \
+#   "paths=./..." \
+#   "output:dir=./../client"
+# popd
 
-go install "${CODEGEN_PKG}"/cmd/openapi-gen
+# go install "${CODEGEN_PKG}"/cmd/openapi-gen
```

Comment:
Won't we want this, once we have some API content?

Reply:
I uncommented this line in latest commit